examples/scenarioSolarArrayDeployment.py

```diff
@@ -0,0 +1,657 @@
+    plt.title(r'Array 2 Element Angle Rates', fontsize=16)
+    plt.ylabel('(deg/s)', fontsize=14)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.legend(loc='center left', prop={'size': 12})
+    plt.grid(True)
+
+    # Plot r_BN_N
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, r_BN_N[:, 0], label=r'$r_{1}$')
+    plt.plot(timespan, r_BN_N[:, 1], label=r'$r_{2}$')
+    plt.plot(timespan, r_BN_N[:, 2], label=r'$r_{3}$')
+    plt.title(r'${}^\mathcal{N} r_{\mathcal{B}/\mathcal{N}}$ Spacecraft Inertial Trajectory', fontsize=16)
+    plt.ylabel('(m)', fontsize=14)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.legend(loc='upper left', prop={'size': 12})
+    plt.grid(True)
+
+    # Plot sigma_BN
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, sigma_BN[:, 0], label=r'$\sigma_{1}$')
+    plt.plot(timespan, sigma_BN[:, 1], label=r'$\sigma_{2}$')
+    plt.plot(timespan, sigma_BN[:, 2], label=r'$\sigma_{3}$')
+    plt.title(r'$\sigma_{\mathcal{B}/\mathcal{N}}$ Spacecraft Inertial MRP Attitude', fontsize=16)
+    plt.ylabel('', fontsize=14)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.legend(loc='lower left', prop={'size': 12})
+    plt.grid(True)
+
+    # Plot omega_BN_B
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, omega_BN_B[:, 0], label=r'$\omega_{1}$')
+    plt.plot(timespan, omega_BN_B[:, 1], label=r'$\omega_{2}$')
+    plt.plot(timespan, omega_BN_B[:, 2], label=r'$\omega_{3}$')
+    plt.title(r'Spacecraft Hub Angular Velocity ${}^\mathcal{B} \omega_{\mathcal{B}/\mathcal{N}}$', fontsize=16)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.ylabel('(deg/s)', fontsize=14)
+    plt.legend(loc='lower right', prop={'size': 12})
+    plt.grid(True)
+
+    # Plot omega_BN_B norm
+    omega_BN_BNorm = np.linalg.norm(omega_BN_B, axis=1)
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, omega_BN_BNorm)
+    plt.title('Hub Angular Velocity Norm $|{}^\mathcal{B} \omega_{\mathcal{B}/\mathcal{N}}|$', fontsize=16)
+    plt.ylabel(r'(deg/s)', fontsize=14)
+    plt.xlabel(r'(min)', fontsize=14)
+    plt.grid(True)
+
+    # Plotting: Conservation quantities
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, rotAngMom_N[:, 1] - rotAngMom_N[0, 1],
+             timespan, rotAngMom_N[:, 2] - rotAngMom_N[0, 2],
+             timespan, rotAngMom_N[:, 3] - rotAngMom_N[0, 3])
+    plt.title('Rotational Angular Momentum Difference', fontsize=16)
+    plt.ylabel('(Nms)', fontsize=14)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.grid(True)
+
+    plt.figure()
+    plt.clf()
+    plt.plot(timespan, rotEnergy[:, 1] - rotEnergy[0, 1])
+    plt.title('Total Energy Difference', fontsize=16)
+    plt.ylabel('Energy (J)', fontsize=14)
+    plt.xlabel('Time (min)', fontsize=14)
+    plt.grid(True)
+
+    if show_plots:
+        plt.show()
+    plt.close("all")
+
+if __name__ == "__main__":
+    run(
+        True,   # show_plots
+    )
+    
```

Comment:
:small_red_triangle_down: Need a new line at the end of the file.

Reply:
Done